File: pset6/mario.py
from cs50 import get_int

height = -1
while height < 1 or height > 8:
    height = get_int("Height: ")
 
x = height 
for i in range(height + 1):
    if i > 0:
        for k in range(x - 1):
            print(" ", end="")
        x -= 1
    for j in range(i):
        print("#", end="")
    if i > 0:
        print()
    
        
        

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    int height = -1;
    while(height > 8 || height < 1)
    {
        height = get_int("Height: ");
    }
    
    int space = height - 1;
    
    for(int x = 0; x < height; x++)
    {
                
        for(int y = space; 0 < y; y--)
        {
            printf(" ");
        }
        space -= 1;
    
        for(int y = x; y >= 0; y--)
        {
            printf("#");
        }
        printf("\n");
    }

}